mobify/platform-scaffold
# [IM Extension] custom commands

This PR adds a custom command that checks if a users email address is in use already to the platform scaffold. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-78
 **Linked PRs**: n/a

## Changes
- Moved isEmailAvailable merlins command to the connector extension
- Removed isEmailAvailable empty command from the SFCC connector

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to the cart and proceed to the shipping page of the checkout
- enter an email address that is already being used for an account
- the login form should show along with a message asking the user to log in
- go back to the email address field and enter an email that's not being used for an account
- the login form and notification should disappear
